src/main/scala/org/fortysevendeg/sparksftp/ReadingSFTPHadoopApp.scala

```diff
@@ -22,6 +25,13 @@ object ReadingSFTPHadoopApp extends IOApp {
       defaultSparkConf: SparkConf = createSparkConfWithSFTPSupport(config)
         .set("fs.sftp.impl", "org.apache.hadoop.fs.sftpwithseek.SFTPFileSystem")
         .set("fs.sftp.impl.disable.cache", "true")
+
+      /**
+        We could also use:
+          .set("spark.master", "local")
+          .set("spark.master", "local[*]")
+        or to run the application locally, for example, through "sbt run"
+       */
```

Comment:
Is this comment related with the `run`. I'd move it above the `def run` to at least have as a scaladoc

Reply:
Changed in a54ef8ff72e58a7412f8dde34c798e0d114f9747